data/json/monsters/feral_humans.json

```
@@ -74,5 +74,125 @@
     "melee_dice": 3,
     "melee_dice_sides": 8,
     "death_drops": "feral_humans_death_drops_axe"
+  },
+  {
+    "id": "mon_feral_scientist_scalpel",
+    "type": "MONSTER",
+    "name": { "str": "mad scientist" },
+    "description": "A researcher who has stared too long into the abyss, and now shambles about muttering nonsense under their breath.  For some reason, the zombies don't seem to mind that this person is obviously still alive.  They're clutching a tiny but razor-sharp blade in one shaky hand.",
+    "default_faction": "science",
+    "looks_like": "chud",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "magenta",
+    "aggression": 30,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 1,
+    "melee_dice_sides": 3,
+    "melee_cut": 2,
+    "dodge": 1,
+    "harvest": "CBM_SCI_FERAL",
+    "vision_day": 50,
+    "vision_night": 3,
+    "death_drops": "feral_scientists_death_drops_scalpel",
+    "death_function": [ "NORMAL" ],
+    "zombify_into": "mon_zombie_scientist",
+    "anger_triggers": [ "FRIEND_DIED", "FRIEND_ATTACKED", "HURT" ],
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "HUMAN",
+      "CAN_OPEN_DOORS",
+      "PATH_AVOID_DANGER_2",
+      "DROPS_AMMO",
+      "CBM_SCI"
+    ]
+  },
+  {
+    "id": "mon_feral_labsecurity_9mm",
+    "type": "MONSTER",
+    "name": { "str": "feral security guard" },
+    "description": "This security guard still breathes, but has been lost to some terrible madness.  They move among the undead, handgun at the ready and saucer pupils scanning for threats in a mockery of their former duty.",
+    "default_faction": "science",
+    "looks_like": "chud",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "magenta",
+    "aggression": 30,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "armor_bash": 4,
+    "armor_cut": 6,
+    "armor_bullet": 4,
+    "armor_stab": 4,
+    "dodge": 1,
+    "harvest": "human",
+    "vision_day": 50,
+    "vision_night": 4,
+    "path_settings": { "max_dist": 10 },
+    "diff": 5,
+    "starting_ammo": { "9mm": 5 },
+    "special_attacks": [
+      {
+        "type": "gun",
+        "cooldown": 10,
+        "move_cost": 150,
+        "gun_type": "m9",
+        "ammo_type": "9mm",
+        "fake_skills": [ [ "gun", 1 ], [ "handgun", 1 ] ],
+        "fake_dex": 9,
+        "fake_per": 9,
```

```suggestion
        "fake_skills": [ [ "gun", 1 ], [ "handgun", 2 ] ],
        "fake_dex": 8,
        "fake_per": 10,
```
Security guards get a lot of practice looking around. The handgun up is because, as I understand it, at low levels dex makes more of a difference than per for guns.